CODEOWNERS

```diff
@@ -5,12 +5,16 @@
 * @bajtos @raymondfeng @kjdelisle
 
 packages/authentication/* @bajtos @kjdelisle
+packages/build/* @bajtos @raymondfeng
 packages/cli/* @raymondfeng @kjdelisle @shimks
 packages/context/* @bajtos @raymondfeng  @kjdelisle
 packages/core/* @bajtos @raymondfeng @kjdelisle
 packages/example-getting-started/* @bajtos @kjdelisle
-packages/openapi-spec-builder/*  @bajtos
-packages/openapi-spec/*  @bajtos
+packages/metadata/* @raymondfeng @virkt25
```

Comment:
Why am I listed as a codeowner for `metadata` 😅 ... I've reviewed the code but never worked on the package.

Reply:
Good point! I found your name in the git history (see e.g. [here](https://github.com/strongloop/loopback-next/commits/master/packages/metadata)). I'll remove you, no problem.